google/cloud/bigquery/magics/magics.py

```diff
@@ -286,7 +286,7 @@ def _handle_error(error, destination_var=None):
 
     Args:
         error (Exception):
-            An exception that ocurred during the query exectution.
+            An exception that ocurred during the query execution.
```

Comment:
Change the word `ocurred` to the correct spelling of `occurred`

`An exception that occurred during the query execution.`